app/src/main/java/mod/jbk/export/GetKeyStoreCredentialsDialog.java

```diff
@@ -172,8 +172,8 @@ public boolean isForSigningWithTestkey() {
         /**
          * @return {@link #keyStorePassword}
          */
-        public String getKeyStorePassword() {
-            return keyStorePassword;
+        public static String getKeyStorePassword() {
+            return binding.etKeyPassword.getText().toString();
         }
```

Comment:
It's bad to get string from non static field for static function.

Reply:
didn't notice that sorry